src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+    private final ColorSupplier ColorSupplier = new ColorSupplier();
+
+    public Ball GetRandomBall() {
```

Comment:
Method names should be in camelCase. Instead of 'GetRandomBall', it should be 'getRandomBall'.